packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/tests-summary.component.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ExtensionSlot } from '@openmrs/esm-framework';
+import { RecentOverviewProps } from '@openmrs/esm-patient-test-results-app';
```

Comment:
@FlorianRappl 
Not sure whether this is okay

Reply:
Nope we should not import MFs - not even types from them.